### PR TITLE
feat: add view client charges component

### PR DESCRIPTION
**Pull request by BLasan**
## Description
Added component to view selected client charge with delete and waive flag options.

## Related issues and discussion
#682 

## Screenshots, if any
webapp1

Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

### CODE REVIEW

**Comment by BLasan** [comment]
Travis build failed due to timeout

**Comment by BLasan** [comment]
@abhaychawla  Please review this

File: src/app/clients/clients-view/general-tab/general-tab.component.scss [hunk 1]
```diff
@@ -12,11 +12,14 @@
...
+    }
     .account-action-button{
       min-width: 26px;
       padding: 0 6px;
       margin: 4px;
       line-height: 25px;
     }
   }
-}
+}
```
Comment (karantakalkar) [inline src/app/clients/clients-view/general-tab/general-tab.component.scss:25]: eof

File: src/app/clients/clients-view/general-tab/general-tab.component.ts [hunk 2]
```diff
@@ -27,6 +27,8 @@ export class GeneralTabComponent implements OnInit {
   showClosedLoanAccounts = false;
   showClosedSavingAccounts = false;
   showClosedShareAccounts = false;
+  /** Route Link */
+  routeLink: any;
```
Comment (karantakalkar) [inline src/app/clients/clients-view/general-tab/general-tab.component.ts:31]: use proper variable names

File: src/app/clients/clients-view/view-charge/view-charge.component.html [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+    <mat-card-title> <i class="fa fa-stop" [ngClass]="(!(chargeData.isWaived || chargeData.isPaid))|statusLookup"></i> 
+      {{ chargeData.name }}
+    </mat-card-title>
+
+    <mat-divider></mat-divider>
+
+    <mat-card-content>
+
+      <div fxLayout="row wrap" class="content" style="padding-top:15px">
+
```
Comment (karantakalkar) [inline src/app/clients/clients-view/view-charge/view-charge.component.html:29]: Screenshot from 2020-05-13 12-19-22
make rows look like this

**Comment by karantakalkar** [comment]
Screenshot from 2020-05-13 12-20-38
take refernce from this view

**Comment by BLasan** [comment]
@karantakalkar  Sorry for the dalay. I'll refactor this

**Comment by BLasan** [comment]
> Screenshot from 2020-05-13 12-20-38
> take reference from this view

Can you put the path to this component?

**Comment by karantakalkar** [comment]
https://github.com/openMF/web-app/pull/711 this PR

**Comment by karantakalkar** [comment]
@BLasan i'll check soon I have meeting rn, I have reviewed most of your p2s can you focus on password model ones they look to be merged by today

**Comment by BLasan** [comment]
> @BLasan i'll check soon I have meeting rn, I have reviewed most of your p2s can you focus on password model ones they look to be merged by today

Sure, I'll refactor it soon

**Comment by BLasan** [comment]
@karantakalkar  Please review this

**Comment by karantakalkar** [comment]
@BLasan SOrry for inconvinience but we are keeping all clients view sub-components on hold for now

**Comment by BLasan** [comment]
> @BLasan Sorry for inconvinience but we are keeping all clients view sub-components on hold for now

It's ok. Do I have to refactor this?

**Comment by karantakalkar** [comment]
@BLasan just reolve mcs I'll merge

**Comment by BLasan** [comment]
> @BLasan just resolve mcs I'll merge

Done

**Pull request merged**